src/prefect/tasks/databricks/databricks_hook.py

```diff
@@ -333,6 +335,37 @@ def terminate_cluster(self, json: dict) -> None:
         """
         self._do_api_call(TERMINATE_CLUSTER_ENDPOINT, json)
 
+    def get_job_id_by_name(self, job_name: str, limit: int = 25) -> int:
+        """
+        Retrieves job_id from run_id.
+
+        Args:
+            - job_name (str): Name of the job.
+
+        Returns
+            - int: Job id for given Databricks job.
+        """
+        matching_jobs = []
+        while True:
```

Comment:
Instead of `while True` with a break, can you use a variable to track when the loop should stop? It's more clear what the purpose of the loop is that way. I'd recommend creating a `more_jobs_to_list` variable which starts out as `True` and then is set to `response_payload["has_more"]` on each loop.

Reply:
Thanks again for the review ☺️